src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -64,12 +64,14 @@ const mockStore = configureMockStore([thunk]);
 
 let store: ReturnType<typeof mockStore>;
 
-const renderComponent = (state: typeof mockState = mockState) => {
+const renderComponent = (state: typeof mockState = mockState, search = '') => {
```

Comment:
`const renderComponent = (params: { state: typeof mockState, url: string } = { state: mockState, url: '/' })` ?

That would avoid the awkwardness of having the undefined in the function call (`renderComponent(undefined, '?focus=foo')`).

By the way, why don't we change `?focus=foo` to a more realistic `/genome-browser?focus=foo`?